packages/compass-sidebar/src/components/resize-handle/resize-handle.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './resize-handle.module.less';
+
+const LEFT_ARROW_KEY = 'ArrowLeft';
+const RIGHT_ARROW_KEY = 'ArrowRight';
+
+function ResizeHandle({
+  onMoveRightPressed,
+  onMoveLeftPressed
+}) {
+  return (
+    <div
+      className={styles['resize-handle']}
+      onKeyDown={(e) => {
+        if (e.key === LEFT_ARROW_KEY) {
+          onMoveLeftPressed();
+        } else if (e.key === RIGHT_ARROW_KEY) {
+          onMoveRightPressed();
+        }
+      }}
+      tabIndex={1}
+    />
```

Comment:
If we want to take it one step further, we can even provide some aria labels describing what's going on here (this is how Slack does it):

<img width="616" alt="image" src="https://user-images.githubusercontent.com/5036933/132712464-7dbe3d60-2d89-4e0f-90c8-6dfe83401acf.png">

(this can also be done later when we are moving this to a shared component as you suggested)

Reply:
Thanks for this suggestion. Definitely would be nice to do. I think we can do it when we move it to a shared place (and use it with the mongosh resizer as well). I'll create a ticket for it.

Reply:
Ticket: https://jira.mongodb.org/browse/COMPASS-5084